sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
   compile project(":sdks:java:extensions:protobuf")
   compile library.java.avro
   compile library.java.bigdataoss_util
+  compile library.java.gax
```

Comment:
As BigQueryServicesImpl uses gax's `com.google.api.gax.core.FixedCredentialsProvider` (undeclared-but-used dependency), declaring it explicitly.